src/apps/vpn/platforms/android/androidutils.cpp

```diff
@@ -29,7 +29,35 @@ constexpr auto UTILS_CLASS = "org/mozilla/firefox/vpn/qt/VPNUtils";
 }  // namespace
 
 // static
-QString AndroidUtils::GetDeviceName() {
+QString AndroidUtils::getDeviceName() {
+  QJniEnvironment env;
+  jclass BUILD = env->FindClass("android/os/Build");
+  jfieldID model = env->GetStaticFieldID(BUILD, "DEVICE", "Ljava/lang/String;");
+  jstring value = (jstring)env->GetStaticObjectField(BUILD, model);
+  if (!value) {
```

Comment:
Consider if value should be renamed to modelName and if buffer should be renamed to something like modelNameBuffer to avoid generic names.